Tasks/AzureStorage/main.ts

```diff
@@ -0,0 +1,45 @@
+import path = require("path");
+import * as os from 'os';
+import util = require("util");
+import tl = require("vsts-task-lib/task");
+import msRestAzure = require("azure-arm-rest/azure-arm-common");
+import armStorage = require("azure-arm-rest/azure-arm-storage");
+import blobUtils = require("./operations/AzureBlobUtils");
+
+function run(): Q.Promise<void> {
+    var artifactsDirectory = tl.getVariable('System.ArtifactsDirectory');
```

Comment:
Currently, the destination for download is "Tfat-agent\_work\r*\a", which is the System.ArtifactsDirectory. Every time a new release happens, the agent deletes folder 'a' and creates a new one. So even though AzCopy does not clean the folder, it works.

Although, I think the usual way, as the agent does download, is to create a folder with name as the artifact alias inside 'a' and download artifacts there. I may change to do that.